src/_cert.ts

```diff
@@ -23,7 +23,7 @@ export interface CommonCertificateOptions {
   organization?: string;
   organizationalUnit?: string;
   emailAddress?: string;
-  domains?: string[];
+  alternativeNames?: string[];
```

Comment:
Since we already released this, we cannot rename an option in non breaking versions. Let's keep `domains` (we might also deprecate and support both)

Reply:
Also genrally, please try to split your pull requests into smaller ones. Like one for CLI arg, one for option rename / tests

Reply:
The renaming is removed. When I split the PR into CLI args and CLI args test, then code coverage checks fails. Should I keep them split anyway?